### DISCUSSION TITLE
OpenModelica Compiler null is Online

### DISCUSSION THREAD

**Discussion by pily1** [OpenModelica/OpenModelica#12442] [Q&A]
hi,I have installed the Eclipse with the version of 4.8 on windows platform,and I have installed the MDT,but the status shows  "OpenModelica Compiler null is Online"
when to startup the Eclipse,it outputs some error,the following image is the message
![1111](https://github.com/OpenModelica/OpenModelica/assets/20216269/f95f938f-7bf8-408c-bf38-1042117f7d99)
how to solve the problem?
think you!

**Comment by adeas31** [comment]
You need to compile OpenModelica with CORBA i.e., --with-omniORB.
I don't think MDT supports the zeroMQ interface but I am not sure. Its been a while I looked at the MDT code. Maybe @adrpo remembers better?

**Reply by pily1** [reply]
hi,I use the openmodelica office installed package instend of myself compiled library,now it get the same error,only the CORBA disabled disappear,the following image is the error
![2222](https://github.com/OpenModelica/OpenModelica/assets/20216269/f1d82efd-53be-4465-b8d8-363c74f62ed6)

**Reply by pily1** [reply]
@adeas31

**Comment by take-study** [comment]
maybe you can run the `omc` command in shell to ensure your `omc` support `corba`

**Reply by pily1** [reply]
@take-study hi,I have tried it,omc.exe outputs has corba,      
--interactive          Sets the interactive mode for omc.
                               Valid options:
                                * none
                                * corba
                                * tcp
                                * zmq

**Comment by take-study** [comment]
is there a objid file created after you run the command

________________________________

**Reply by pily1** [reply]
@take-study no,there is nothing objid file after I run the command

**Reply by take-study** [reply]
if you post the output which you run the command in shell that will be helpful

**Reply by pily1** [reply]
@take-study following is the omc.exe output
[output.txt](https://github.com/user-attachments/files/15509170/output.txt)

**Comment by adeas31** [comment]
It should create a file `openmodelica.objid.mdt` in your temp folder.

What is the output if you call,

`omc.exe -c=mdt +d=interactiveCorba`

**Comment by pily1** [comment]
I have not solved it,I don't use the Eclipse now,I add log in the code ,then I analysis log to get the code processing
